[2025-04-13T20:24:15.986+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T20:24:16.013+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.process_data scheduled__1989-01-01T00:00:00+00:00 [queued]>
[2025-04-13T20:24:16.029+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.process_data scheduled__1989-01-01T00:00:00+00:00 [queued]>
[2025-04-13T20:24:16.030+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-04-13T20:24:16.051+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_data> on 1989-01-01 00:00:00+00:00
[2025-04-13T20:24:16.060+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gcs_dlt_data_pipeline', 'process_data', 'scheduled__1989-01-01T00:00:00+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/first_stage/first_stage.py', '--cfg-path', '/tmp/tmp7c7vokeq']
[2025-04-13T20:24:16.063+0000] {standard_task_runner.py:105} INFO - Job 240: Subtask process_data
[2025-04-13T20:24:16.069+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=735) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-13T20:24:16.070+0000] {standard_task_runner.py:72} INFO - Started process 737 to run task
[2025-04-13T20:24:16.210+0000] {task_command.py:467} INFO - Running <TaskInstance: gcs_dlt_data_pipeline.process_data scheduled__1989-01-01T00:00:00+00:00 [running]> on host 7f7577d1d274
[2025-04-13T20:24:16.324+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_dlt_data_pipeline' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='1989-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1989-01-01T00:00:00+00:00'
[2025-04-13T20:24:16.325+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-13T20:24:37.897+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-04-13T20:24:38.135+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T20:24:38.137+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
